### DISCUSSION TITLE
[Feature request] Redirect alias to different address after certain time

### DISCUSSION THREAD

**Discussion by grzegor** [anonaddy/anonaddy#377] [Feature Requests]
This is what I feel a very common case. Let's say you need to register on some random forum just to be able to download an attachment. You don't care about it at all after the activation email arrives. It would be great if during the alias creation I could set a time after a chosen action would be taken:
- emails would be forwarded to my different email address (a spam holder) instead of my main mailbox, or
- alias would get deactivated
For example, I set that after 1h (or 1 day, 1 week) after creation the "Recipients" record would change from my main address to my spam address (which could also be a `mainaddress+label@domain.tld` if someone just wants to flag them and their email provider accepts that format like on Gmail or Office 365).

**Comment by willbrowningme** [comment]
I'm not sure I'll add the "change recipient" after a certain period of time but I will try to add the "deactivate" after a certain time - https://github.com/anonaddy/anonaddy/issues/252
